mistletoe/html_attributes_renderer.py

```diff
@@ -0,0 +1,201 @@
+"""
+HTML Attributes renderer for mistletoe.
+"""
+
+import html
+from mistletoe import block_token
+from mistletoe import span_token
+from mistletoe.block_token import HTMLAttributes
+from mistletoe.html_renderer import HTMLRenderer
+
+
+class HTMLAttributesRenderer(HTMLRenderer):
+    """
+    HTML Attributes renderer class.
+
+    See mistletoe.html_renderer module for more info.
+    """
+    def __init__(self):
```

Comment:
Suggest to take an "extras" parameter like the HTMLRenderer does, so that this renderer too can be extended.

Reply:
See (c6a9ed2a70268a73adb5ce4d24d567418b2e2efa)